test/uk/gov/hmrc/nisp/views/HtmlSpec.scala

```diff
@@ -0,0 +1,181 @@
+/*
+ * Copyright 2017 HM Revenue & Customs
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package uk.gov.hmrc.nisp.views.html
+
+
+
+import org.jsoup.Jsoup
+import org.jsoup.nodes.{Document, Element}
+import play.api.i18n.Messages
+import org.scalatest._
+import org.scalatest.mock.MockitoSugar
+import org.scalatestplus.play.OneAppPerSuite
+import play.api.http.Status
+import play.api.test.FakeRequest
+import play.api.test.Helpers._
+import uk.gov.hmrc.http.cache.client.SessionCache
+import uk.gov.hmrc.nisp.config.ApplicationConfig
+import uk.gov.hmrc.nisp.connectors.NispConnector
+import uk.gov.hmrc.nisp.controllers.connectors.CustomAuditConnector
+import uk.gov.hmrc.nisp.helpers._
+import uk.gov.hmrc.nisp.models.{SPAmountModel, StatePensionAmount, StatePensionAmountRegular}
+import uk.gov.hmrc.nisp.services.CitizenDetailsService
+import uk.gov.hmrc.play.frontend.auth.{AuthContext, AuthenticationProviderIds, LoggedInUser, Principal}
+import uk.gov.hmrc.play.frontend.auth.connectors.AuthConnector
+import uk.gov.hmrc.play.http.SessionKeys
+import uk.gov.hmrc.play.partials.CachedStaticHtmlPartialRetriever
+import uk.gov.hmrc.play.test.UnitSpec
+import uk.gov.hmrc.time.DateTimeUtils.now
+import play.twirl.api.Html
+import org.apache.commons.lang3.StringEscapeUtils
+import uk.gov.hmrc.nisp.controllers.auth.NispUser
+import uk.gov.hmrc.play.frontend.auth.connectors.domain.{Accounts, ConfidenceLevel, CredentialStrength, PayeAccount}
+
+trait HtmlSpec extends UnitSpec  {
```

Comment:
Can we reformat this document?

In IntelliJ you can use `Code -> Refomat Code` or `Ctrl` + `Alt` + `L`